components/prism-hpkp.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Original by Scott Helme.
+ *
+ * Reference: https://scotthelme.co.uk/hpkp-cheat-sheet/
+ */
+
+Prism.languages.hpkp = {
+	'directive':  {
+             pattern: /\b(?:max-age=|includeSubDomains|preload|strict|report-uri=|report-to|pin-sha256=)/,
+             alias: 'keyword'
+        },
+	'safe': {
+            pattern: /([0-9]{7,})/,
```

Comment:
Parentheses are not needed around the whole pattern. Same goes for the regexp below and the others in the hsts component. ;)